### PR TITLE
Add CREATE_EVENTS permission

### PR DESCRIPTION
**Pull request by jocafamaka**
New permission `CREATE_EVENTS` not yet documented but already out.

Discussion: 
- https://github.com/discord/discord-api-docs/issues/6298

### CODE REVIEW

**Comment by SQKo** [comment]
The `create_events` value is wrong, it is 44 not 43.
Also please put in the correct order of the bits

File: src/Discord/Parts/Permissions/ChannelPermission.php [hunk 1]
```diff
@@ -25,7 +25,8 @@
  * @property bool $move_members             Allows for moving of members between voice channels
  * @property bool $use_vad                  Allows for using voice-activity-detection in a voice channel
  * @property bool $request_to_speak         Allows for requesting to speak in stage channels. (*This permission is under active development and may be changed or removed.*)
- * @property bool $manage_events            Allows for creating, editing, and deleting scheduled events
+ * @property bool $create_events            Allows for creating scheduled events
```
Comment (SQKo) [inline src/Discord/Parts/Permissions/ChannelPermission.php:28]: please also move this between use_soundboard and use_external_sounds

File: src/Discord/Parts/Permissions/RolePermission.php [hunk 2]
```diff
@@ -38,6 +38,7 @@
  * @property bool $use_vad
  * @property bool $request_to_speak
  * @property bool $manage_events
+ * @property bool $create_events
```
Comment (SQKo) [inline src/Discord/Parts/Permissions/RolePermission.php:41]: please also move this between use_soundboard and use_external_sounds

**Review by SQKo — CHANGES_REQUESTED** [review]

**Comment by key2peace** [comment]
[this link](https://github.com/discord/discord-api-docs/issues/6298#issuecomment-1637297533) also talks about the requirement to add the ``x-super-properties header`` header to your requests in order to support it, which is not present [here](https://github.com/discord-php/DiscordPHP-Http/blob/master/src/Discord/Http.php#L266-L276)

**Comment by SQKo** [comment]
> [this link](https://github.com/discord/discord-api-docs/issues/6298#issuecomment-1637297533) also talks about the requirement to add the `x-super-properties` header to your requests in order to support it, which is not present [here](https://github.com/discord-php/DiscordPHP-Http/blob/master/src/Discord/Http.php#L266-L276)

That is not something the bot usually use, the header is sent by discord clients

**Comment by jocafamaka** [comment]
It appers the permissions requirements will not affect bots, but it still aply to roles and users.

https://github.com/discord/discord-api-docs/commit/8d9f1fb17458f7372c1a2c6b029e397483c8be2f

**Review by SQKo — APPROVED** [review]
need to test

**Pull request merged**